### PR TITLE
chore: rebuilding the requirements with new constraint

### PR DESCRIPTION
**Pull request by deborahgu**
setup.py has a `load_requirements` method which gives `install_requires` a list of packages to install. The logic was not respecting constraints. However, we have modified this method elsewhere in the code base to respect constraints, so I updated  accordingly.

### CODE REVIEW

**Comment by justinhynes** [comment]
I _think_ we still want to bump the version here too.

File: i18n/__init__.py [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
 class Runner:
     """
     Runner class for internationalization.
     """
+
     def __init__(self):
         self.args = sys.argv[1:]
         self.configuration = None
         self.parser = argparse.ArgumentParser()
-        self.parser.add_argument(
```
Comment (deborahgu) [inline i18n/__init__.py:20]: everything from here on down was just automatic lint.

File: setup.py [hunk 2]
```diff
@@ -1,44 +1,81 @@
 #!/usr/bin/env python
 import os
-import re
 import pathlib
+import re
 
 from setuptools import setup
 
 
-def load_requirements(*requirements_paths):
```
Comment (deborahgu) [inline setup.py:9]: taken from https://github.com/openedx/xblock-sdk/blob/master/setup.py. everything below this method is just linter.

**Review by justinhynes — APPROVED** [review]
Oh wow, good find on the setup.py needing to be updated.

**Pull request merged**